### PR TITLE
Separated the copy component and added functionality to copy code in the docs section

### LINKED ISSUE #39: Add Copy Button in the Docs Section
Currently when a user tries to install the npm package, the code has to be copied at all like this.

https://user-images.githubusercontent.com/67755381/153450046-3148e35f-dc91-4206-ad44-a13989539a19.mp4

Instead we can add a copy button to it like the one as shown so that the user can copy it without having selecting all.
image_2022-02-10_215001

### PR DESCRIPTION
**Pull request by AnshumanDhiman**
Fixed #39 Created a new PR, the old PR 42  was creating issues of conflicting, as discussed with @kbhutani0001 I have created a new file.

### CODE REVIEW

**Comment by vinayaksh42** [comment]
@AnshumanDhiman is this PR same as https://github.com/EOS-uiux-Solutions/eos-icons-landing/pull/42?

**Comment by AnshumanDhiman** [comment]
yes sir both the PR's are same, I was facing conflicting issues with the old PR and after force push it didn't meet up the expected changes. I had a conversation with Kartikey sir regarding this issue to open a new PR regarding this issue.

**Comment by vinayaksh42** [comment]
> yes sir both the PR's are same, I was facing conflicting issues with the old PR and after force push it didn't meet up the expected changes. I had a conversation with Kartikey sir regarding this issue to open a new PR regarding this issue.

Can you close your previous PR if it is no longer required?

**Comment by AnshumanDhiman** [comment]
sure sir

File: src/pages/Docs.js [hunk 1]
```diff
@@ -491,7 +579,16 @@ const Docs = () => {
                 </a>
               </h3>
               <pre className='code language-shell'>
-                <code>npm install eos-icons-react</code>
+                <code className='copytext12 code-style'>
```
Comment (cyntss) [inline src/pages/Docs.js:582]: We have a coding convention for these type of scenarios where the class is merely used as an identifier for JS, but has no CSS styling at all. In these scenarios, we use the naming convention:
`js-NAME`

So, could you please add the prefix `js-` to all these `copytextNN` classes?

**Comment by cyntss** [comment]
@AnshumanDhiman always think about the scalability of your applications. All solutions need to be thought to not introduce human error when maintenance, new releases, new features come into play.

Make sure your components are ready to be re-used. Make sure your data has 1 true source. Consider new actors that don't know the codebase and how they will scale or integrate with your applications.

**Comment by AnshumanDhiman** [comment]
As I am having conflict issues here, I think I must close this PR and create a new PR on this. Thanks for helping😀

**Pull request closed without merging**